pyproject.toml

```diff
@@ -36,6 +37,7 @@ remote-processors = { path = "lib/remote-processors", develop = true }
 integration = { path = "apps/integration", develop = true }
 sycamore-poetry-lock = { path = "lib/poetry-lock", develop = true }
 aryn-sdk = { path = "lib/aryn-sdk", develop = true }
+queryui = { path = "apps/query-ui", develop = true }
```

Comment:
I don't think this dependency needs to be here, right? queryui depends on sycamore, not the other way around.

Reply:
If it's not there, when you do a poetry install in the root you don't get the stuff to run the query ui.